### PR TITLE
DATA: Add Created & Updated Dates to Comment

### LINKED ISSUE #77: DATA: Add Created & Updated Dates to Comment

### PR DESCRIPTION
**Pull request by cjdutoit**
closes #77

### CODE REVIEW

File: Taarafo.Core/Migrations/20211111212024_ExtendCommentModel.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
```
Comment (hassanhabib) [inline Taarafo.Core/Migrations/20211111212024_ExtendCommentModel.cs:1]: Add a header to this file.

**Comment by ShriHumrudha** [comment]
Why is the actual model file not in this PR?

File: Taarafo.Core/Migrations/20211111212024_ExtendCommentModel.cs (hunk 1 above)
Comment (cjdutoit) [reply Taarafo.Core/Migrations/20211111212024_ExtendCommentModel.cs]: @hassanhabib, I have updated this

**Comment by cjdutoit** [comment]
> Why is the actual model file not in this PR?

It is already part of master via #66.  My bad.  Still getting used to the different commit process.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**